daemon/pool.go

```diff
@@ -32,6 +36,31 @@ var (
 	errDriverStopped = errors.NewKind("driver stopped")
 )
 
+var (
+	policyDefaultWindow    = envIntOr("BBLFSHD_POLICY_WINDOW", 30)
```

Comment:
What are the units on this? (And perhaps more importantly, how will the user know?)

Reply:
Changed units, it's now a time interval, instead of multiple of tick rate.